location_field/static/location_field/js/form.js

```diff
@@ -158,6 +185,10 @@
                     });
                 },
 
+                yandexSearchProvider: function (options, onload) {
+                    onload();
```

Comment:
Can we make sure it's possible to pass at least some **options**? To name a few that **I think** would be useful for someone: `apiKey`, `lang`, `kind` (not 100% sure about this one)?